lib/rubocop/cop/rspec/multiple_describes.rb

```diff
@@ -36,6 +101,59 @@ def on_top_level_group(node)
 
           add_offense(node.send_node)
         end
+
+        def on_send(node)
+          return unless style == :splitting
+
+          return unless describe?(node)
+
+          return if includes_common_codes?(node)
+
+          block_node = node.parent
+          return if root_node?(block_node)
+          return if includes_describe_in_previous_node?(block_node)
+
+          add_offense(node, message: MSG_SPLIT)
+        end
+
+        private
+
+        def includes_describe_in_previous_node?(block_node)
+          previous_block_node = previous_sibling(block_node)
+          return false if previous_block_node.nil?
+
+          if previous_block_node.child_nodes.any? { |child| describe?(child) }
+            return true
+          end
+
+          includes_describe_in_previous_node?(previous_block_node)
+        end
```

Comment:
It's hard to grasp. Is it checking all previous example groups to have a nested `describe` in them?

Reply:
I'm sorry for the confusing code.
Yes, I want to check all previous example groups.
Therefore, I changed to a shorter code than the previous one.